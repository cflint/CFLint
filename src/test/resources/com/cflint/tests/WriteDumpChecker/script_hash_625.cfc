<cfscript>
#writeDump(now(), 'HH:mm:ss')#
</cfscript>