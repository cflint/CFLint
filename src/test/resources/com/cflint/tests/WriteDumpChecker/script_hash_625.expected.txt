{
  "version" : "",
  "timestamp" : 1568688625,
  "issues" : [ {
    "severity" : "INFO",
    "id" : "AVOID_USING_WRITEDUMP",
    "message" : "AVOID_USING_WRITEDUMP",
    "category" : "CFLINT",
    "abbrev" : "AU",
    "locations" : [ {
      "file" : "src\\test\\resources\\com\\cflint\\tests\\WriteDumpChecker\\script_hash_625.cfc",
      "fileName" : "script_hash_625.cfc",
      "function" : "",
      "offset" : 12,
      "column" : 2,
      "line" : 2,
      "message" : "Avoid using the writeDump function in production code.",
      "variable" : "writeDump",
      "expression" : "writeDump(now(), 'HH:mm:ss')"
    } ]
  } ],
  "counts" : {
    "totalFiles" : 0,
    "totalLines" : 0,
    "countByCode" : [ {
      "code" : "AVOID_USING_WRITEDUMP",
      "count" : 1
    } ],
    "countBySeverity" : [ {
      "severity" : "INFO",
      "count" : 1
    } ]
  }
}